### ISSUE TITLE
Huge lag being caused on my server (Updated SuperbVote to latest version)

### ISSUE THREAD

**Issue by CaptainAmpersand** [Rsl1122/Plan-PlayerAnalytics#532]
### Server Information
**Plan Version:** 4.1.6
**Server:** CraftBukkit 1.12.2-R0.1-SNAPSHOT git-Spigot-4bd94dc-aa34e9b (MC: 1.12.2)
**Database:** SQLite schema v13

**Operating SubSystem:** Linux (amd64) version 4.14.14-nfo-xen
**Java Version:** 1.8.0_152, Oracle Corporation
**Java VM Version:** Java HotSpot(TM) 64-Bit Server VM version 25.152-b16, Oracle Corporation
**Java VM Flags:** [-XX:MaxMetaspaceSize=256M, -Xmx11972M]
**Connection Log:**

Server Address | Request Type | Response | Sent
-- | -- | -- | --

**Servers:**

Server Name | Address | UUID 
-- | -- | --
NanoNuke | http://go.nanonuke.net:8804 | cf21f04c-5556-3ac3-82b8-dd60f621cd14
### Logged Errors
```
java.sql.SQLException: database connection closed
   org.sqlite.core.CoreConnection.checkOpen(CoreConnection.java:336)
   org.sqlite.jdbc4.JDBC4Connection.prepareStatement(JDBC4Connection.java:45)
   org.sqlite.jdbc3.JDBC3Connection.prepareStatement(JDBC3Connection.java:263)
   org.sqlite.jdbc3.JDBC3Connection.prepareStatement(JDBC3Connection.java:235)
   main.java.com.djrapitops.plan.database.tables.Table.query(Table.java:214)
   main.java.com.djrapitops.plan.database.tables.IPsTable.getGeoInfo(IPsTable.java:71)
   main.java.com.djrapitops.plan.database.tables.IPsTable.saveGeoInfo(IPsTable.java:92)
   main.java.com.djrapitops.plan.systems.processing.player.IPUpdateProcessor.process(IPUpdateProcessor.java:36)
   main.java.com.djrapitops.plan.systems.queue.ProcessConsumer.consume(ProcessingQueue.java:55)
   main.java.com.djrapitops.plan.systems.queue.ProcessConsumer.consume(ProcessingQueue.java:40)
   main.java.com.djrapitops.plan.systems.queue.Consumer.run(Consumer.java:34)
   com.djrapitops.plugin.task.RunnableFactory$1.run(RunnableFactory.java:38)
   org.bukkit.craftbukkit.v1_12_R1.scheduler.CraftTask.run(CraftTask.java:71)
   org.bukkit.craftbukkit.v1_12_R1.scheduler.CraftAsyncTask.run(CraftAsyncTask.java:52)
   java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
   java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
   java.lang.Thread.run(Thread.java:748)
```
- [x] ~~Fixed~~ Can not be fixed because cause is unknown (Likely cause: Reload)
```
java.sql.SQLException: The database has been closed
   org.sqlite.core.NativeDB.throwex(NativeDB.java:471)
   org.sqlite.core.NativeDB.prepare_utf8(Native Method)
   org.sqlite.core.NativeDB.prepare(NativeDB.java:127)
   org.sqlite.core.DB.prepare(DB.java:227)
   org.sqlite.core.CorePreparedStatement.(CorePreparedStatement.java:41)
   org.sqlite.jdbc3.JDBC3PreparedStatement.(JDBC3PreparedStatement.java:30)
   org.sqlite.jdbc4.JDBC4PreparedStatement.(JDBC4PreparedStatement.java:19)
   org.sqlite.jdbc4.JDBC4Connection.prepareStatement(JDBC4Connection.java:48)
   org.sqlite.jdbc3.JDBC3Connection.prepareStatement(JDBC3Connection.java:263)
   org.sqlite.jdbc3.JDBC3Connection.prepareStatement(JDBC3Connection.java:235)
   main.java.com.djrapitops.plan.database.tables.Table.query(Table.java:214)
   main.java.com.djrapitops.plan.database.tables.WorldTable.getWorldNames(WorldTable.java:139)
   main.java.com.djrapitops.plan.utilities.analysis.AnalysisUtils.addMissingWorlds(AnalysisUtils.java:145)
   main.java.com.djrapitops.plan.utilities.html.structure.SessionTabStructureCreator.createStructure(SessionTabStructureCreator.java:87)
   main.java.com.djrapitops.plan.utilities.html.HtmlStructure.createSessionsTabContentInspectPage(HtmlStructure.java:70)
   main.java.com.djrapitops.plan.systems.info.parsing.InspectPageParser.parse(InspectPageParser.java:129)
   main.java.com.djrapitops.plan.systems.info.parsing.InspectPageParser.parse(InspectPageParser.java:73)
   main.java.com.djrapitops.plan.systems.info.BukkitInformationManager.getPlayerHtml(BukkitInformationManager.java:266)
   main.java.com.djrapitops.plan.systems.webserver.response.InspectPageResponse.(InspectPageResponse.java:23)
   main.java.com.djrapitops.plan.systems.info.BukkitInformationManager.lambda$cachePlayer$0(BukkitInformationManager.java:120)
   main.java.com.djrapitops.plan.systems.webserver.pagecache.PageCache.cachePage(PageCache.java:96)
   main.java.com.djrapitops.plan.systems.info.BukkitInformationManager.cachePlayer(BukkitInformationManager.java:118)
   main.java.com.djrapitops.plan.systems.cache.SessionCache.endSession(SessionCache.java:55)
   main.java.com.djrapitops.plan.systems.processing.player.EndSessionProcessor.process(EndSessionProcessor.java:28)
   main.java.com.djrapitops.plan.systems.queue.ProcessConsumer.consume(ProcessingQueue.java:55)
   main.java.com.djrapitops.plan.systems.queue.ProcessConsumer.consume(ProcessingQueue.java:40)
   main.java.com.djrapitops.plan.systems.queue.Consumer.run(Consumer.java:34)
   com.djrapitops.plugin.task.RunnableFactory$1.run(RunnableFactory.java:38)
   org.bukkit.craftbukkit.v1_12_R1.scheduler.CraftTask.run(CraftTask.java:71)
   org.bukkit.craftbukkit.v1_12_R1.scheduler.CraftAsyncTask.run(CraftAsyncTask.java:52)
   java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
   java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
   java.lang.Thread.run(Thread.java:748)
```
- [x] ~~Fixed~~ Can not be fixed because cause is unknown (Likely cause: Reload)
```
java.lang.IllegalStateException: Duplicate key 1296
   java.util.stream.Collectors.lambda$throwingMerger$0(Collectors.java:133)
   java.util.HashMap.merge(HashMap.java:1254)
   java.util.stream.Collectors.lambda$toMap$58(Collectors.java:1320)
   java.util.stream.ReduceOps$3ReducingSink.accept(ReduceOps.java:169)
   java.util.stream.ReferencePipeline$2$1.accept(ReferencePipeline.java:175)
   java.util.stream.ReferencePipeline$2$1.accept(ReferencePipeline.java:175)
   java.util.ArrayList$ArrayListSpliterator.forEachRemaining(ArrayList.java:1382)
   java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:481)
   java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:471)
   java.util.stream.ReduceOps$ReduceOp.evaluateSequential(ReduceOps.java:708)
   java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234)
   java.util.stream.ReferencePipeline.collect(ReferencePipeline.java:499)
   com.djrapitops.pluginbridge.plan.griefprevention.GriefPreventionData.getPlayerData(GriefPreventionData.java:42)
   com.djrapitops.plan.data.plugin.HookHandler.getInspectContainersFor(HookHandler.java:96)
   com.djrapitops.plan.system.webserver.response.pages.parts.InspectPagePluginsContent.generateForThisServer(InspectPagePluginsContent.java:42)
   com.djrapitops.plan.system.info.request.GenerateInspectPluginsTabRequest.generateAndCache(GenerateInspectPluginsTabRequest.java:61)
   com.djrapitops.plan.system.info.request.GenerateInspectPluginsTabRequest.runLocally(GenerateInspectPluginsTabRequest.java:67)
   com.djrapitops.plan.system.info.BukkitInfoSystem.runLocally(BukkitInfoSystem.java:32)
   com.djrapitops.plan.system.info.connection.ConnectionSystem.sendInfoRequest(ConnectionSystem.java:80)
   com.djrapitops.plan.system.info.connection.BukkitConnectionSystem.lambda$sendWideInfoRequest$2(BukkitConnectionSystem.java:91)
   com.djrapitops.plan.system.info.connection.WebExceptionLogger.logIfOccurs(WebExceptionLogger.java:24)
   com.djrapitops.plan.system.info.connection.BukkitConnectionSystem.sendWideInfoRequest(BukkitConnectionSystem.java:89)
   com.djrapitops.plan.system.info.request.GenerateInspectPageRequest.generateAndCache(GenerateInspectPageRequest.java:66)
   com.djrapitops.plan.system.info.request.GenerateInspectPageRequest.runLocally(GenerateInspectPageRequest.java:75)
   com.djrapitops.plan.system.info.BukkitInfoSystem.runLocally(BukkitInfoSystem.java:32)
   com.djrapitops.plan.system.info.InfoSystem.sendRequest(InfoSystem.java:97)
   com.djrapitops.plan.system.info.InfoSystem.generateAndCachePlayerPage(InfoSystem.java:63)
   com.djrapitops.plan.system.cache.SessionCache.lambda$endSession$3(SessionCache.java:75)
   com.djrapitops.plan.system.info.connection.WebExceptionLogger.logIfOccurs(WebExceptionLogger.java:24)
   com.djrapitops.plan.system.cache.SessionCache.endSession(SessionCache.java:75)
   com.djrapitops.plan.system.processing.processors.player.EndSessionProcessor.process(EndSessionProcessor.java:28)
   com.djrapitops.plan.system.processing.ProcessConsumer.consume(ProcessingQueue.java:80)
   com.djrapitops.plan.system.processing.ProcessConsumer.consume(ProcessingQueue.java:66)
   com.djrapitops.plan.utilities.queue.Consumer.run(Consumer.java:34)
   com.djrapitops.plugin.task.RunnableFactory$1.run(RunnableFactory.java:39)
   org.bukkit.craftbukkit.v1_12_R1.scheduler.CraftTask.run(CraftTask.java:71)
   org.bukkit.craftbukkit.v1_12_R1.scheduler.CraftAsyncTask.run(CraftAsyncTask.java:52)
   java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
   java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
   java.lang.Thread.run(Thread.java:748)
```
- [ ] Fixed
```
java.lang.NoSuchMethodError: io.minimum.minecraft.superbvote.storage.VoteStorage.getVotes(Ljava/util/UUID;)I
   com.djrapitops.pluginbridge.plan.superbvote.SuperbVoteData.getServerData(SuperbVoteData.java:47)
   com.djrapitops.plan.utilities.analysis.Analysis.lambda$analyzeAdditionalPluginData$4(Analysis.java:128)
   java.util.stream.ForEachOps$ForEachOp$OfRef.accept(ForEachOps.java:184)
   java.util.ArrayList$ArrayListSpliterator.forEachRemaining(ArrayList.java:1382)
   java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:481)
   java.util.stream.ForEachOps$ForEachTask.compute(ForEachOps.java:291)
   java.util.concurrent.CountedCompleter.exec(CountedCompleter.java:731)
   java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:289)
   java.util.concurrent.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1056)
   java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1692)
   java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:157)
```
- [x] Fixed
```
java.lang.NoSuchMethodError: io.minimum.minecraft.superbvote.storage.VoteStorage.getVotes(Ljava/util/UUID;)I
   com.djrapitops.pluginbridge.plan.superbvote.SuperbVoteData.getPlayerData(SuperbVoteData.java:35)
   com.djrapitops.plan.data.plugin.HookHandler.getInspectContainersFor(HookHandler.java:96)
   com.djrapitops.plan.system.webserver.response.pages.parts.InspectPagePluginsContent.generateForThisServer(InspectPagePluginsContent.java:42)
   com.djrapitops.plan.system.info.request.GenerateInspectPluginsTabRequest.generateAndCache(GenerateInspectPluginsTabRequest.java:61)
   com.djrapitops.plan.system.info.request.GenerateInspectPluginsTabRequest.runLocally(GenerateInspectPluginsTabRequest.java:67)
   com.djrapitops.plan.system.info.BukkitInfoSystem.runLocally(BukkitInfoSystem.java:32)
   com.djrapitops.plan.system.info.connection.ConnectionSystem.sendInfoRequest(ConnectionSystem.java:80)
   com.djrapitops.plan.system.info.connection.BukkitConnectionSystem.lambda$sendWideInfoRequest$2(BukkitConnectionSystem.java:91)
   com.djrapitops.plan.system.info.connection.WebExceptionLogger.logIfOccurs(WebExceptionLogger.java:24)
   com.djrapitops.plan.system.info.connection.BukkitConnectionSystem.sendWideInfoRequest(BukkitConnectionSystem.java:89)
   com.djrapitops.plan.system.info.request.GenerateInspectPageRequest.generateAndCache(GenerateInspectPageRequest.java:66)
   com.djrapitops.plan.system.info.request.GenerateInspectPageRequest.runLocally(GenerateInspectPageRequest.java:75)
   com.djrapitops.plan.system.info.BukkitInfoSystem.runLocally(BukkitInfoSystem.java:32)
   com.djrapitops.plan.system.info.InfoSystem.sendRequest(InfoSystem.java:97)
   com.djrapitops.plan.system.info.InfoSystem.generateAndCachePlayerPage(InfoSystem.java:63)
   com.djrapitops.plan.system.cache.SessionCache.lambda$null$1(SessionCache.java:58)
   com.djrapitops.plan.system.info.connection.WebExceptionLogger.logIfOccurs(WebExceptionLogger.java:24)
   com.djrapitops.plan.system.cache.SessionCache.lambda$cacheSession$2(SessionCache.java:57)
   com.djrapitops.plan.system.processing.ProcessConsumer.consume(ProcessingQueue.java:80)
   com.djrapitops.plan.system.processing.ProcessConsumer.consume(ProcessingQueue.java:66)
   com.djrapitops.plan.utilities.queue.Consumer.run(Consumer.java:34)
   com.djrapitops.plugin.task.RunnableFactory$1.run(RunnableFactory.java:39)
   org.bukkit.craftbukkit.v1_12_R1.scheduler.CraftTask.run(CraftTask.java:71)
   org.bukkit.craftbukkit.v1_12_R1.scheduler.CraftAsyncTask.run(CraftAsyncTask.java:52)
   java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
   java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
   java.lang.Thread.run(Thread.java:748)
```
- [x] Fixed
```
java.lang.NoSuchMethodError: io.minimum.minecraft.superbvote.storage.VoteStorage.getVotes(Ljava/util/UUID;)I
   com.djrapitops.pluginbridge.plan.superbvote.SuperbVoteData.getPlayerData(SuperbVoteData.java:35)
   com.djrapitops.plan.data.plugin.HookHandler.getInspectContainersFor(HookHandler.java:96)
   com.djrapitops.plan.system.webserver.response.pages.parts.InspectPagePluginsContent.generateForThisServer(InspectPagePluginsContent.java:42)
   com.djrapitops.plan.system.info.request.GenerateInspectPluginsTabRequest.generateAndCache(GenerateInspectPluginsTabRequest.java:61)
   com.djrapitops.plan.system.info.request.GenerateInspectPluginsTabRequest.runLocally(GenerateInspectPluginsTabRequest.java:67)
   com.djrapitops.plan.system.info.BukkitInfoSystem.runLocally(BukkitInfoSystem.java:32)
   com.djrapitops.plan.system.info.connection.ConnectionSystem.sendInfoRequest(ConnectionSystem.java:80)
   com.djrapitops.plan.system.info.connection.BukkitConnectionSystem.lambda$sendWideInfoRequest$2(BukkitConnectionSystem.java:91)
   com.djrapitops.plan.system.info.connection.WebExceptionLogger.logIfOccurs(WebExceptionLogger.java:24)
   com.djrapitops.plan.system.info.connection.BukkitConnectionSystem.sendWideInfoRequest(BukkitConnectionSystem.java:89)
   com.djrapitops.plan.system.info.request.GenerateInspectPageRequest.generateAndCache(GenerateInspectPageRequest.java:66)
   com.djrapitops.plan.system.info.request.GenerateInspectPageRequest.runLocally(GenerateInspectPageRequest.java:75)
   com.djrapitops.plan.system.info.BukkitInfoSystem.runLocally(BukkitInfoSystem.java:32)
   com.djrapitops.plan.system.info.InfoSystem.sendRequest(InfoSystem.java:97)
   com.djrapitops.plan.system.info.InfoSystem.generateAndCachePlayerPage(InfoSystem.java:63)
   com.djrapitops.plan.system.cache.SessionCache.lambda$endSession$3(SessionCache.java:75)
   com.djrapitops.plan.system.info.connection.WebExceptionLogger.logIfOccurs(WebExceptionLogger.java:24)
   com.djrapitops.plan.system.cache.SessionCache.endSession(SessionCache.java:75)
   com.djrapitops.plan.system.processing.processors.player.EndSessionProcessor.process(EndSessionProcessor.java:28)
   com.djrapitops.plan.system.processing.ProcessConsumer.consume(ProcessingQueue.java:80)
   com.djrapitops.plan.system.processing.ProcessConsumer.consume(ProcessingQueue.java:66)
   com.djrapitops.plan.utilities.queue.Consumer.run(Consumer.java:34)
   com.djrapitops.plugin.task.RunnableFactory$1.run(RunnableFactory.java:39)
   org.bukkit.craftbukkit.v1_12_R1.scheduler.CraftTask.run(CraftTask.java:71)
   org.bukkit.craftbukkit.v1_12_R1.scheduler.CraftAsyncTask.run(CraftAsyncTask.java:52)
   java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
   java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
   java.lang.Thread.run(Thread.java:748)
```
- [x] Fixed
### Benchmarks
```
192 ms    Analysis: 3rd party
355 ms    Analysis: Data Analysis
104 ms    Analysis: Fetch Phase
5 ms      Analysis: Source AdvancedAchievements
12 ms     Analysis: Source AdvancedAntiCheat
187 ms    Analysis: Source Economy (Essentials Economy)
187 ms    Analysis: Source Essentials
8 ms      Analysis: Source GriefPrevention
12 ms     Analysis: Source Jobs
21 ms     Analysis: Source MCMMO
22 ms     Analysis: Source SuperbVote
12 ms     Analysis: Source ViaVersion
662 ms    Analysis: Total
1635 ms   Enable
160 ms    Init Database
15 ms     Initializing locale
30 ms     Inspect Parse, Fetch
15 ms     Processed BanAndOpProcessor.
11 ms     Processed BukkitConnectionSystem$$Lambda$193/213443332.
2 ms      Processed CommandProcessor.
83 ms     Processed EndSessionProcessor.
175 ms    Processed IPUpdateProcessor.
125 ms    Processed InfoSystem$$Lambda$197/136633083.
58 ms     Processed KickProcessor.
0 ms      Processed KillProcessor.
4 ms      Processed NameProcessor.
104 ms    Processed NetworkPageUpdateProcessor.
1 ms      Processed NetworkSettings$$Lambda$201/445520376.
4 ms      Processed PlayerHackKickListener$$Lambda$521/1612041437.
17 ms     Processed PlayerVersionListener$$Lambda$424/1792052828.
12 ms     Processed RegisterProcessor.
123 ms    Processed SessionCache$$Lambda$421/254329161.
18 ms     Processed TPSInsertProcessor.
11 ms     Register Listeners
1 ms      Task Registration
58 ms     WebServer Initialization
```
### Debug Log
```
You're using the latest version.
Using Locale: Default (EN)
Database init..
SQLite database: Opened a new Connection
Started task 182: DBConnectionPingTask SQLite, Async.
Started task 183: DB Clean Task, Async.
sqlite-database connection established.
Initializing Webserver..
No Certificate -> Using Http server for Visualization.
Https Start Failed.
Webserver running on PORT 8804 (http://go.nanonuke.net:8804)
Started task 184: ProcessQueueConsumer, Async.
Started task 185: ProcessQueueConsumer, Async.
Started task 186: ProcessQueueConsumer, Async.
Started task 187: ProcessQueueConsumer, Async.
Started task 188: ProcessQueueConsumer, Async.
Started task 189: ProcessQueueConsumer, Async.
Started task 190: ProcessQueueConsumer, Async.
Started task 191: ProcessQueueConsumer, Async.
Analysis | Boot analysis in 30 seconds..
Started task 192: TPSCountTimer, Sync.
Started task 193: NetworkPageRefreshTask, Async.
Started task 194: BootAnalysisTask, Async.
Started task 195: PeriodicAnalysisTask, Async.
Registered a new datasource: AdvancedAntiCheat
Registered a new datasource: AdvancedAchievements
Registered a new datasource: Essentials
Registered a new datasource: GriefPrevention
Registered a new datasource: Jobs
Registered a new datasource: MCMMO
Registered a new datasource: SuperbVote
Registered a new datasource: Economy (Essentials Economy)
Registered a new datasource: ViaVersion
Verbose debug messages are enabled.
|                  | Enable (Feb 21, 00:01:04) ------------
| Feb 21, 00:01:04 | 15 ms     Initializing locale
| Feb 21, 00:01:04 | 160 ms    Init Database
| Feb 21, 00:01:04 | 58 ms     WebServer Initialization
| Feb 21, 00:01:04 | 11 ms     Register Listeners
| Feb 21, 00:01:04 | 1 ms      Task Registration
| Feb 21, 00:01:04 | Enabling bStats Metrics.
| Feb 21, 00:01:04 | 1635 ms   Enable
| ---------------- | --------------------------------------
Player Analytics Enabled.
NetworkSettings: Fetch Config settings from database..
NetworkSettings: No Config settings in database.
Ended task 183: DB Clean Task, Async.
Analysis | Starting Boot Analysis..
Analysis | Beginning analysis of user data..
Analysis | Fetching Data..
Analysis | Data Fetched (106 users, took 104ms), beginning Analysis of data..
Analysis | Analyzing additional data sources (3rd party)
|                  | Analysis (Feb 21, 00:01:37) ------------
| Feb 21, 00:01:37 | Analysis Fetch Phase
| Feb 21, 00:01:37 | started   Analysis: Fetch Phase
| Feb 21, 00:01:38 | 104 ms    Analysis: Fetch Phase
| Feb 21, 00:01:38 | started   Analysis: Data Analysis
| Feb 21, 00:01:38 | Analysis Phase
| Feb 21, 00:01:38 | 355 ms    Analysis: Data Analysis
| Feb 21, 00:01:38 | Analyzing additional data sources (3rd party)
| Feb 21, 00:01:38 | started   Analysis: 3rd party
| Feb 21, 00:01:38 | Additional Sources: 9
| Feb 21, 00:01:38 | started   Analysis: Source MCMMO
| Feb 21, 00:01:38 | started   Analysis: Source GriefPrevention
| Feb 21, 00:01:38 | started   Analysis: Source AdvancedAchievements
| Feb 21, 00:01:38 | started   Analysis: Source SuperbVote
| Feb 21, 00:01:38 | started   Analysis: Source Economy (Essentials Economy)
| Feb 21, 00:01:38 | started   Analysis: Source Jobs
| Feb 21, 00:01:38 | started   Analysis: Source AdvancedAntiCheat
| Feb 21, 00:01:38 | started   Analysis: Source ViaVersion
| Feb 21, 00:01:38 | started   Analysis: Source Essentials
| Feb 21, 00:01:38 | 5 ms      Analysis: Source AdvancedAchievements
| Feb 21, 00:01:38 | 8 ms      Analysis: Source GriefPrevention
| Feb 21, 00:01:38 | 12 ms     Analysis: Source Jobs
| Feb 21, 00:01:38 | 12 ms     Analysis: Source ViaVersion
| Feb 21, 00:01:38 | 12 ms     Analysis: Source AdvancedAntiCheat
| Feb 21, 00:01:38 | 21 ms     Analysis: Source MCMMO
| Feb 21, 00:01:38 | 22 ms     Analysis: Source SuperbVote
| Feb 21, 00:01:38 | 187 ms    Analysis: Source Economy (Essentials Economy)
| Feb 21, 00:01:38 | 187 ms    Analysis: Source Essentials
| Feb 21, 00:01:38 | 192 ms    Analysis: 3rd party
| Feb 21, 00:01:38 | 662 ms    Analysis: Total
| ---------------- | ----------------------------------------
Analysis | Analysis Complete. (took 662 ms) 
Ended task 194: BootAnalysisTask, Async.
First run of TPSCountTimer Task.
```
### config.yml
```
# -----------------------------------------------------
# Plan Bukkit Configuration file
# More information about each setting:
# https://github.com/Rsl1122/Plan-PlayerAnalytics/wiki/Bukkit-Configuration
# -----------------------------------------------------
Server:
    ServerName: 'NanoNuke'
# -----------------------------------------------------
# More information about Locale
# https://github.com/Rsl1122/Plan-PlayerAnalytics/wiki/Localization
# -----------------------------------------------------
Plugin:
    Debug: false
    Locale: default
    WriteNewLocaleFileOnEnable: false
    Bungee-Override:
        StandaloneMode: false
        CopyBungeeConfig: true
    Dev:
# -----------------------------------------------------
# More information about SSL Certificate Settings:
# https://github.com/Rsl1122/Plan-PlayerAnalytics/wiki/SSL-Certificate-%28HTTPS%29-Set-Up
# -----------------------------------------------------
WebServer:
    Port: 8804
    # InternalIP usually does not need to be changed, only change it if you know what you're doing!
    # 0.0.0.0 allocates Internal (local) IP automatically for the WebServer.
    InternalIP: 0.0.0.0
    Security:
        SSL-Certificate:
            KeyStorePath: 'SSLCertificate.keystore'
            Alias: 'alias'
    # For those that want to serve Html from their own WebServer instead.
    # Set up Html Export (https://github.com/Rsl1122/Plan-PlayerAnalytics/wiki/External-WebServer-Use)
    # ATTENTION: On BungeeCord systems it is not possible to disable the WebServer on the plugin due to WebAPI requirements.
    # If the WebServer is disabled with this setting BungeeCord systems will cease to function.
    DisableWebServer: false
    ExternalWebServerAddress: "go.nanonuke.net"
# -----------------------------------------------------
Database:
    Type: SQLite
    MySQL:
        Host: localhost
        Port: 3306
        User: root
        Database: Plan
        # Launch options to append after driver address, for example
        LaunchOptions: "?rewriteBatchedStatements=true&useSSL=false"
# -----------------------------------------------------
Commands:
    AlternativeIP:
        Enabled: true
        Link: go.nanonuke.net:%port%
    Colors:
        Main: '&2'
        Secondary: '&7'
        Highlight: '&f'
Analysis:
    AutoRefreshPeriod: 60
    Active:
        # Minutes a player should play per week to be considered active
        PlaytimeThreshold: 30
        # How many days player should join per week to be considered active
        LoginThreshold: 2
    LogProgress: true
    Export:
        Enabled: false
        DestinationFolder: 'Analysis Results'
Data:
    Commands:
        LogUnknownCommands: false
        CombineCommandAliases: true
# -----------------------------------------------------
Customization:
    UseServerTime: true
    Display:
        SessionsAsTable: false
        LargestWorldPercInSessionTitle: false
        # By Default WorldPie is ordered alphabetically, colors are still determined alphabetically.
        OrderWorldPieByPercentage: false
        MaxSessions: 50
        MaxPlayers: 2500
        MaxPlayersPlayersPage: 25000
        PlayerTableFooter: true
    Formatting:
        DecimalPoints: '#.##'
        TimeAmount:
            Year: '1 year, '
            Years: '%years% years, '
            Month: '1 month, '
            Months: '%months% months, '
            Day: '1d '
            Days: '%days%d '
            Hours: '%hours%h '
            Minutes: '%minutes%m '
            Seconds: '%seconds%s'
            Zero: '0s'
        # Dates settings use Java SimpleDateFormat.
        # You can find the patterns & examples here:
        # https://docs.oracle.com/javase/7/docs/api/java/text/SimpleDateFormat.html
        Dates:
            # RecentDays replaces date with Today, Yesterday, Wednesday etc.
            RecentDays: true
                # Non-regex pattern to replace
                DatePattern: 'MMM d YYYY'
            Full: 'MMM d YYYY, HH:mm:ss'
            NoSeconds: 'MMM d YYYY, HH:mm'
            JustClock: 'HH:mm:ss'
    WorldAliases:
        world: world
        Resworld: Resworld
        Spawn: Spawn
        world_creative: world_creative
        Testing: Testing
        world_nether: world_nether
        world_the_end: world_the_end
# -----------------------------------------------------
# More information about Themes:
# https://github.com/Rsl1122/Plan-PlayerAnalytics/wiki/Themes
#
# Since 4.1.0 colors are set in theme.yml
# -----------------------------------------------------
Theme:
    Base: Default
    Graphs:
        TPS:
            High-Threshold: 18
            Medium-Threshold: 10
# -----------------------------------------------------
Plugins:
    AdvancedAchievements:
        Enabled: true
    AdvancedAntiCheat:
        Enabled: true
    Economy (Essentials Economy):
        Enabled: true
    Essentials:
        Enabled: true
    Factions:
        HideFactions:
            - ExampleFaction
    GriefPrevention:
        Enabled: true
    Jobs:
        Enabled: true
    MCMMO:
        Enabled: true
    SuperbVote:
        Enabled: true
    Towny:
        HideTowns:
            - ExampleTown
    ViaVersion:
        Enabled: true
    BuyCraft:
```

**Comment by Rsl1122** [comment]
Try changing this to false:
```
SuperbVote:
        Enabled: true
```

**Comment by CaptainAmpersand** [comment]
Buuut.. I want it to be tracked D:
(lol, will do now.)
Mind you SuperbVote just came out with a new update.

**Comment by Rsl1122** [comment]
Most of the errors fixed in commit ec6fb4106e217f22b40c52f09858363e52fb9d05
